app/code/Magento/Downloadable/Test/Mftf/Test/VerifyDisableDownloadableProductSamplesAreNotAccessibleTest.xml

```diff
@@ -101,14 +101,14 @@
         </actionGroup>
 
         <!-- Link Sample url is not accessible. You are redirected to Home Page -->
-        <seeInCurrentUrl url="{{StorefrontHomePage.url}}" stepKey="seeRedirectToHomePage"/>
+        <seeCurrentUrlEquals url="http://{$hostname}/" stepKey="seeRedirectToHomePage"/>
 
         <!-- Navigate to Sample url on Storefront -->
         <actionGroup ref="StorefrontOpenDownloadableSampleActionGroup" stepKey="openDownloadableSample">
             <argument name="sampleId" value="{$grabDownloadableSampleId}"/>
         </actionGroup>
 
         <!-- Sample url is not accessible. You are redirected to Home Page -->
-        <seeInCurrentUrl url="{{StorefrontHomePage.url}}" stepKey="seeHomePage"/>
+        <seeCurrentUrlEquals url="http://{$hostname}/" stepKey="seeHomePage"/>
```

Comment:
```suggestion
        <seeCurrentUrlEquals url="{{_ENV.MAGENTO_ BASE_URL}}" stepKey="seeHomePage"/>
```

Reply:
I have updated this still getting something error with this

Reply:
Please remove space there`{{_ENV.MAGENTO_BASE_URL}}`

Reply:
Done